### PR TITLE
Add writeJson method to 'http' module

### PR DESCRIPTION
**Pull request by daaAd1**
Solving this [issue](https://github.com/storyscript/runtime/issues/626) .

### CODE REVIEW

File: microservice.yml [hunk 1]
```diff
@@ -93,6 +93,19 @@ actions:
                   type: boolean
                   in: responseBody
                   required: false
+            writeJSON:
+              http: &response_http
```
Comment (wilzbach) [inline microservice.yml]: You copied too much here. You redefine the YAML key.
What you want is:

```yaml
              http:
                <<: *response_http
```

**Comment by daaAd1** [comment]
Thanks for feedback, should be fixed now. Let me know if you need anything else.

**Comment by wilzbach** [comment]
FYI: I added two small follow-up commits as I didn't want to bother you with this (https://github.com/storyscript/http/pull/55/commits/a062958871e8684d51cefa55bf84b084d74d86ef and  https://github.com/storyscript/http/pull/55/commits/ac005e39c177caa42de41aee322e36ee6f139acc). I hope you don't mind.

**Comment by daaAd1** [comment]
Yes, of course, no problem.

**Pull request merged**